src/main/java/subsystems/gyroscope/Gyroscope.java

```diff
@@ -1,9 +1,9 @@
-package frc.robot.subsystems.gyroscope;
+package subsystems.gyroscope;
 
 import com.kauailabs.navx.frc.AHRS;
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.wpilibj.SPI;
-import frc.robot.subsystems.LoggedSubsystem;
+import subsystems.LoggedSubsystem;
```

Comment:
These are not changes that should be introduced in this branch.

Reply:
OK

Reply:
So, are you going to do something to fix the issue?

Also, please note you aren't supposed to resolve conversations made on your own PRs. It's the PR author's job to fix the issue, but the reviewer's job to check the fix was good.